Gemfile

```diff
@@ -52,6 +52,9 @@ gem 'request_store_rails'
 gem 'sidekiq'
 gem 'clockwork'
 
+# For outgoing http requests
+gem 'http'
```

Comment:
We might already have a transient dependency on Faraday in our stack due to using `json_api_client`; this is not a suggestion to switch to that, just ℹ️

Reply:
ok thanks, Tijmen suggested we include the http gem